docs/Consensus-Protocols/IBFT.md

```diff
@@ -0,0 +1,117 @@
+description: Pantheon IBFT 2.0 Proof-of-Authority (PoA) consensus protocol implementation
+<!--- END of page meta data -->
+
+# IBFT 2.0
+
+Pantheon implements the IBFT 2.0 Proof-of-Authority (PoA) consensus protocol. IBFT 2.0 can be used for private networks. 
+
+In IBFT 2.0 networks, transactions and blocks are validated by approved accounts, known as validators. 
+Validators take turns to create the next block. Existing validators propose and vote to add or remove validators. 
+
+## Genesis File
+
+To use IBFT 2.0 requires an IBFT 2.0 genesis file. The genesis file defines properties specific to IBFT 2.0:
+
+!!! example "IBFT 2.0 Genesis File (stripped)"
+    ```json
+      {
+        "config": {
+          ...
+          "revisedibft": {
+            "blockperiodseconds": 2,
+            "epochlength": 30000,
+            "requesttimeoutseconds": 10
+          }
+        },
+        "nonce": "0x0",
+        "extraData": "0xf853a00000000000000000000000000000000000000000000000000000000000000000ea94be068f726a13c8d46c44be6ce9d275600e1735a4945ff6f4b66a46a2b2310a6f3a93aaddc0d9a1c193808400000000c0",
+        "difficulty": "0x1",
+        "mixHash": "0x63746963616c2062797a616e74696e65206661756c7420746f6c6572616e6365",
+        ...
+      }
+    ```
+    
+Properties specific to IBFT 2.0 are:
+
+* `blockperiodseconds` - Minimum block time in seconds. 
+* `epochlength` - Number of blocks after which to reset all votes.
+* `requesttimeoutseconds` - Timeout for each consensus round before a round change. 
+* `extraData` - `RLP([32 Bytes Vanity, List<Validators>, No Votes, Round=Int(0), 0 Seals])`
+
+The `extraData` property is RLP (Recursive Length Prefix) encoded. RLP encoding is a space efficient object 
```

Comment:
Example of usage of the abbreviation extension
However, it's better to group all abbreviation definitions at the end of the page.
```suggestion
The `extraData` property is RLP encoded. RLP encoding is a space efficient object 
*[RLP]: Recursive Length Prefix
```

Reply:
Done